packages/block-library/src/post-author/edit.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp, useEntityId } from '@wordpress/core-data';
+import { RichText } from '@wordpress/block-editor';
+
+function PostAuthorDisplay() {
+	const [ author ] = useEntityProp( 'postType', 'post', 'author' );
+	return <RichText.Content tagName="h6" value={ String( author ) } />;
```

Comment:
Similar to my question in #19578 , why an h6 tag? These tags have strong semantic meaning for screen readers and will break the post or page hierarchy. Validators will also flag this.

Reply:
Which tag do you suggest?

Reply:
This is what the [address](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address) element is for.

Reply:
Done 👍